lighthouse-core/test/lib/minification-estimator-test.js

```diff
@@ -227,5 +227,19 @@ describe('minification estimator', () => {
       // Already-minified source script. estimated 1% smaller minified
       expect(minificationPct).toBeCloseTo(0.01);
     });
+
+    it('should handle nested template literals', () => {
+      // Basic nested literals
+      const nestedTemplates = 'window.myString=`foo${` bar ${` baz ${` bam `} `} `} `';
+      expect(computeJSTokenLength(nestedTemplates)).toEqual(nestedTemplates.length);
+
+      // Can get rid of 5 spaces after normal code braces
+      const nestedWithCode = 'window.myString=`foo${` bar ${{}     }`}`';
+      expect(computeJSTokenLength(nestedWithCode)).toEqual(nestedWithCode.length - 5);
+
+      // Ignore braces in string
+      const nestedTemplatesBrace = 'window.myString=`{foo${` }bar ${` baz ${` bam `} `} `} `';
```

Comment:
nice cases! maybe just one more that flips back and forth between brace types?

```js
window.myString=`${({foo: bar.map(() => ({baz: `${'}'}`}))})}`

Reply:
Is this test code fully minified?

Reply:
no there should be 4 spaces that can be removed

Reply:
I removed the spaces for this case, would it be better to keep them an adjust the expectation?

Reply:
yeah I would probably prefer that so it recognizes it's in JS

Reply:
Not a big deal though, so I'll leave this up to you @adamraine if you want to address before merge